projects/laji/src/app/shared-modules/observation-result/selected-field-group/selected-field-group.component.ts

```diff
@@ -76,12 +79,14 @@ export class SelectedFieldGroupComponent {
   }
 
   getFieldColumnArray(field: string) {
-    if (/gathering\.conversions\.(wgs84|euref|ykj)(CenterPoint)$/.test(field)) {
+    if (!field) {
+      return;
+    } else if (/gathering\.conversions\.(wgs84|euref|ykj)(CenterPoint)/.test(field)) {
       return [
         field + '.lat',
         field + '.lon'
       ];
-    } else if (/gathering\.conversions\.(wgs84|euref|ykj)$/.test(field)) {
+    } else if (/gathering\.conversions\.(wgs84|euref|ykj)/.test(field)) {
```

Comment:
Why the `$` check was removed from the two regexps? It's a good check, if we e.g. add a new field `gathering.conversions.wgs84.CenterPoint.SomeSubProp`, that would pass the first check. `$` means that the checked string must end there.

Reply:
Is because otherwise that function can't be used in getFieldColumnArrayLength, as the field name there has  .lat .long etc. ends. Although I could modify the regexp so that it detects either field names ending in (wgs84|euref|ykj), or it accepts lat/lon properties after it, so that the firs one could be someting like `/gathering\.conversions\.(wgs84|euref|ykj)(CenterPoin)($|.(lat|lon)$)`.

Reply:
How about `/gathering\.conversions\.(wgs84|euref|ykj)CenterPoint(lat|lon)?$/`

Reply:
Sent changes, it now works as the old one, and it produces the column array for a member of those same columns.

Reply:
@olzraiti simplified the regexp using your suggestion.